src/components/Content/Content.js

```diff
@@ -1,15 +1,214 @@
-import React from "react";
+import React, { useState } from "react";
+import DataService from "../../services/DataService";
+import { FormGroup, Label, Input, InputGroup, InputGroupAddon, InputGroupText } from "reactstrap";
 
 import "./Content.css";
 
+const formatSections = (data, activeSection, activeSubsection, setActiveSubsection, variables, setVariables) => {
+  if (data == null) {
+    return;
+  }
+  return Object.entries(data).map(section => {
+    const key = section[0];
+    const value = section[1];
```

Comment:
Reads better if you reduce this to a single line
```suggestion
    const [key, value] = section;
```